doc/Architecture/arc-tec_x-road_technologies.md

```diff
@@ -0,0 +1,137 @@
+# X-Road technologies
+
+**Technical Specification**
+
+Version: 0.1
+18.01.2018
+<!-- 3 pages -->
+Doc. ID: ARC-TEC
+
+---
+
+## Version history
+
+ Date       | Version | Description                                                 | Author
+ ---------- | ------- | ----------------------------------------------------------- | --------------------
+ 18.01.2018 | 0.1     | Initial version                                             | Antti Luoma
```

Comment:
Should the version be 1.0 instead? The other docs start from 0.1 but assuming they were incrementally written along with development their draft versioning makes more sense than using it here. The content is copied from documents that are past the draft stage after all. Admittedly the convention on versioning is not exactly clear.